src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReevaluateTest.java

```diff
@@ -26,7 +26,8 @@
 import static org.junit.Assert.*;
 
 @RunWith(Parameterized.class)
```

Comment:
I suggest to delete this test class or just add the `@Ignore` tag on top of the class. It is just a suggestion, what do you think @m-brophy

Reply:
Agreed, I've deleted the test class,the reevaluate agenda, the constant declaration and a couple of other classes